package.json

```diff
@@ -9,6 +9,7 @@
     "body-parser": "^1.20.2",
     "commander": "^10.0.0",
     "express": "^4.18.2",
-    "mjml": "^4.14.0"
+    "mjml": "^4.14.0",
+    "newrelic": "^9.15.0"
```

Comment:
Should we add a comma to the end of this, so next time we don't hijack the ownership of this line?

Reply:
unfortunately can't do that on a JSON file